### PR TITLE
Use alternative background color for line highlighting in dark mode

### LINKED ISSUE #2412: Use alternative background color for line highlighting in dark mode
The yellow background color that is used for line highlighting makes the actual code hard to read in dark mode:

image

This is not the case in light mode:

image

Example: [submission 6623077](https://dodona.ugent.be/en/submissions/6623077/)

### PR DESCRIPTION
**Pull request by TimonDB**
This pull request adapts the color for highlighting code lines in dark mode in case of a runtime error. The code line should now be more readable.

Before:
Screenshot from 2021-08-03 12-07-59

After:
Screenshot from 2021-08-03 14-41-54

Closes #2412 .

### CODE REVIEW

**Review by chvp — COMMENTED** [review]
@bmesuere probably has a better opinion on this, but it still doesn't seem very readable to me. Going (a lot) darker might be better.

**Comment by bmesuere** [comment]
> @bmesuere probably has a better opinion on this, but it still doesn't seem very readable to me. Going (a lot) darker might be better.

Indeed, I would also create a separate variable for this since the warning color can be used in other places and this change can thus have other unintended side effects.

**Review by chvp — APPROVED** [review]

**Review by niknetniko — APPROVED** [review]

File: app/assets/stylesheets/theme/theme-dark.css.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@
   $danger-800: $red-200 !global;
   $danger-500: $red-300 !global;
   $info-500: $light-blue-300 !global;
-  $warning-500: $amber-300 !global;
```
Comment (chvp) [inline app/assets/stylesheets/theme/theme-dark.css.scss:26]: Ah, actually, this probably shouldn't be changed. AFAIK this is used in a number of other places, where we don't necessarily want to change the color. If this messes up the left border you can create an extra variable for it.

Comment (TimonDB) [reply app/assets/stylesheets/theme/theme-dark.css.scss:26]: Yes, I just saw it too. I will undo it

Comment (TimonDB) [reply app/assets/stylesheets/theme/theme-dark.css.scss:26]: It has minimal effect on the left border when I change it back to `amber-300`, so creating an extra variable for the left border is not necessary I think 
Screenshot from 2021-08-03 14-54-00

**Comment by TimonDB** [comment]
As a result, in light theme there is also a left border visible, can this stay this way?
Screenshot from 2021-08-03 14-58-50

**Review by chvp — APPROVED** [review]

**Review by bmesuere — APPROVED** [review]

**Pull request merged**